src/rdkafka_cgrp.h

```diff
@@ -183,8 +182,10 @@ typedef struct rd_kafka_cgrp_s {
 
         int32_t            rkcg_generation_id;      /* Current generation id */
 
-        rd_kafka_assignor_t *rkcg_assignor;         /* Selected partition
-                                                     * assignor strategy. */
+        rd_kafka_assignor_t *rkcg_assignor;         /**< The current partition
+                                                     *   assignor. */
+        void              *rkcg_assignor_state;     /**< current partition
```

Comment:
Is the assignor state needed when not the group leader? If not, put this in rkcg_group_leader

Reply:
it's not needed by the leader, only the members.